docs/data/room.md

```diff
@@ -0,0 +1,36 @@
+# Room
+
+A Room is an abstract concept that the player exists in and interacts with.
+
+- A Room has a unique [room type](./roomType.md)
+- An instance of a Room is a [RoomState](./roomState.md)
+- A Room is defined by a state machine
+  - states are [PlayerState](./playerState.md)s
+  - transitions are [Command](./command.md)s
+  - the state machine is modeled by a [RoomHandler](../architecture/roomHandler.md)
+- A Room has one entrance ([Door](./door.md))
+- A Room has one or more exits (Doors)
```

Comment:
should this link to `./door.md`?

Reply:
Nope. I was trying to link the first reference of any keyword within the doc, so that there's only one link per reference.